ruma-events-macros/src/gen.rs

```diff
@@ -269,42 +214,19 @@ impl ToTokens for RumaEvent {
                 TokenStream::new()
             };
 
-        let stripped_fields = event_fields
-            .iter()
-            .map(|event_field| strip_serde_attrs(event_field));
-
+        let event_type_name = self.event_type.value();
         let output = quote!(
             #(#attrs)*
-            #[derive(Clone, PartialEq, Debug, ruma_events_macros::FromRaw)]
+            #[derive(serde::Serialize, Clone, PartialEq, Debug, ruma_events_macros::FromRaw)]
```

Comment:
I'd like to have the builtin derives first.

```suggestion
            #[derive(Clone, PartialEq, Debug, serde::Serialize, ruma_events_macros::FromRaw)]
```